### DISCUSSION TITLE
4.26 packaging error?

### DISCUSSION THREAD

**Discussion by gradgr** [TheHerobrine/DarkerNodes#29] [General]
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: begin: stack for UAT
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: === Critical error: ===
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error:
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: Assertion failed: IsValid() [File:D:\Build\++UE4\Sync\Engine\Source\Runtime\Core\Public\Templates/SharedPointer.h] [Line: 879]
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error:
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error:
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error:
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeb3a8d759 KERNELBASE.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f78c386 UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f78fae8 UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f49dd5d UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f435615 UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f4376c0 UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3ecb4f65 UE4Editor-Slate.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe1561b884 UE4Editor-DarkerNodes.dll!FDarkerNodesModule::StartupModule() [D:\Build\++Portal+Promotion\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DarkerNodes\Source\DarkerNodes\Private\DarkerNodes.cpp:15]
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffe3f501032 UE4Editor-Core.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeadd2e7d3 UE4Editor-Projects.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeadd4354a UE4Editor-Projects.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeadd2e5d0 UE4Editor-Projects.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740f9537b UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740f98a22 UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740f90eac UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740f9121a UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740fa527d UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ff740fa732e UE4Editor-Cmd.exe!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeb53a7034 KERNEL32.DLL!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: [Callstack] 0x00007ffeb633d0d1 ntdll.dll!UnknownFunction []
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error:
UATHelper: Packaging (Android (ASTC)):   LogWindows: Error: end: stack for UAT
PackagingResults: Error: begin: stack for UAT
PackagingResults: Error: === Critical error: ===
PackagingResults: Error: Assertion failed: IsValid() [File:D:\Build\++UE4\Sync\Engine\Source\Runtime\Core\Public\Templates/SharedPointer.h] [Line: 879]
PackagingResults: Error: [Callstack] 0x00007ffeb3a8d759 KERNELBASE.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3f78c386 UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3f78fae8 UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3f49dd5d UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3f435615 UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3f4376c0 UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe3ecb4f65 UE4Editor-Slate.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffe1561b884 UE4Editor-DarkerNodes.dll!FDarkerNodesModule::StartupModule() [D:\Build\++Portal+Promotion\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DarkerNodes\Source\DarkerNodes\Private\DarkerNodes.cpp:15]
PackagingResults: Error: [Callstack] 0x00007ffe3f501032 UE4Editor-Core.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffeadd2e7d3 UE4Editor-Projects.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffeadd4354a UE4Editor-Projects.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffeadd2e5d0 UE4Editor-Projects.dll!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740f9537b UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740f98a22 UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740f90eac UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740f9121a UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740fa527d UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ff740fa732e UE4Editor-Cmd.exe!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffeb53a7034 KERNEL32.DLL!UnknownFunction []
PackagingResults: Error: [Callstack] 0x00007ffeb633d0d1 ntdll.dll!UnknownFunction []
PackagingResults: Error: end: stack for UAT
UATHelper: Packaging (Android (ASTC)): Took 9.2044163s to run UE4Editor-Cmd.exe, ExitCode=3
UATHelper: Packaging (Android (ASTC)): ERROR: Cook failed.

**Comment by hugoattal** [comment]
@gradgr Yes, sorry for this, this just has been fixed and will be available in 1.3 update!
Linked issue: https://github.com/TheHerobrine/DarkerNodes/issues/25

**Comment by gradgr** [comment]
No worries 👍
